### ISSUE TITLE
[Accessibility] Message toast component text is not read by screen reader

### ISSUE THREAD

**Issue by harshitha-s-kanchan** [SAP/fundamental-ngx#11098]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Screen reader should be able to read out toast message. 

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular - 15
Ngx - 0.43.11 (Issue also present in the latest version)
Screen reader - JAWS 2023.2311.34

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

Note - **JWAS Screen reader voice added in the below screen recordings**

https://github.com/SAP/fundamental-ngx/assets/149040487/70b58288-4c63-45c0-a399-944d401ea9d5

https://github.com/SAP/fundamental-ngx/assets/149040487/0baf7089-b28a-4c64-9b6c-a9545ac39b68

https://github.com/SAP/fundamental-ngx/assets/149040487/813e09da-0f34-4267-8f99-53f142ab7b02

#### Please provide relevant source code (if applicable).
https://github.com/SAP/fundamental-ngx/blob/6aa1336396713b3d9be4920e4c6179d530dbef4f/libs/core/message-toast/message-toast.component.ts#L22

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes

**Comment by droshev** [comment]
@nikolay-kolarov Can you advise us here? thanks

**Comment by droshev** [comment]
We need to add `role`
```
<div role='alert'> Message Toast created from component.
                It will stay open if the cursor is above it.
                Has a customized width and will disappear after 7500ms </div>
```

**Comment by mikerodonnell89** [comment]
Closing, this has been fixed:

![Image](https://github.com/user-attachments/assets/a4c02aa8-b4a3-462a-a668-7a60a726b41c)
